Sources/SIL/SILParser.swift

```diff
@@ -182,11 +189,23 @@ class SILParser: Parser {
             let value = try parseInt()
             return .integerLiteral(type, value)
         case "load":
+            let _ = skip("[copy]")
+            let _ = skip("[take]")
+            let _ = skip("[trivial]")
```

Comment:
Let's have these as fields on `load` (and add a test that uses all these flags)?